hasura/migrations/default/1705087600730_migrate_dropped_app_rows_to_app_metadata/down.sql

```diff
@@ -0,0 +1 @@
+-- Not sure there is a down migration for this?
```

Comment:
@andy-t-wang As I can see you are filling empty table, if so, then you can create down migration to remove all rows in the table, just in case.

Reply:
resolved